### PR TITLE
ICDS Branding

### PR DESCRIPTION
**Pull request by sravfeyn**
https://docs.google.com/document/d/1QCUHA7PLvZPspuY23Pn98HIrD4zv_KuakFoJwi1dGiE/edit#

@nickpell @snopoke

### CODE REVIEW

File: corehq/apps/export/templates/export/partial/export_list_controller.html [hunk 1]
```diff
@@ -16,6 +16,8 @@
     {% endif %}
     <p>
         {% block lead_text %}
+        {% blocktrans with hq_name=django_settings.COMMCARE_HQ_NAME %}
+            Exports are a way to download data from {{ hq_name }}
```
Comment (snopoke) [inline corehq/apps/export/templates/export/partial/export_list_controller.html]: missing `endblocktrans`

**Comment by sravfeyn** [comment]
Corresponding ansible change https://github.com/dimagi/commcarehq-ansible/commit/96cd91e1498b06dafcb581c7a4fae9ebe3459b00

File: corehq/util/context_processors.py [hunk 2]
```diff
@@ -87,7 +87,7 @@ def websockets_override(request):
         return {}
 
 
-def enterprise_mode(request):
+def django_settings(request):
     return {
-        'enterprise_mode': settings.ENTERPRISE_MODE
+        'django_settings': settings
```
Comment (snopoke) [inline corehq/util/context_processors.py]: I don't think it's a good idea to expose ALL settings here.

**Review by snopoke — CHANGES_REQUESTED** [review]
Looking good so far

File: corehq/util/context_processors.py (hunk 2 above)
Comment (sravfeyn) [inline corehq/util/context_processors.py]: Why is this bad?

Comment (snopoke) [inline corehq/util/context_processors.py]: It just makes it easier to expose secrets to the world. A typo that got missed in code review could put our entire settings dict into a template. Best to be explicit about what templates should have access to.

**Comment by sravfeyn** [comment]
@snopoke Updated. Except for 1,2,3 and 5 in the spec, everything else is implemented.

**Pull request merged**